src/main/sensors/rangefinder.c

```diff
@@ -85,23 +83,43 @@ PG_RESET_TEMPLATE(sonarConfig_t, sonarConfig,
 );
 #endif
 
+void pgResetFn_rangefinderConfig(rangefinderConfig_t *rangefinderConfig)
+{
+    rangefinderConfig->rangefinder_hardware = RANGEFINDER_NONE;
+
+#if defined(USE_RANGEFINDER_LIDARLITE)
```

Comment:
you don't need reset function for this; designated initializer will work fine in reset template